src/components/ArticleContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+import { getArticleById } from "../api/api";
+import { useParams } from "react-router-dom";
+import ArticleBody from "./ArticleBody";
+
+const ArticleContainer = () => {
+  let { articleID } = useParams();
+
+  const [article, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticleById(articleID).then(({ article }) => {
+      setArticles(article);
+    });
+  }, [articleID]);
+
+  return (
+    <div className="ArticleContainer">
+      <ArticleBody key={article.article_id} article={article} />
```

Comment:
there's no need for a `key` prop here - it's not in an array (you're not using a `.map`!)